packages/nimble-components/src/icon-base/index.ts

```diff
@@ -25,27 +25,6 @@ export class Icon extends FoundationElement {
     public constructor(/** @internal */ public readonly icon: NimbleIcon) {
         super();
     }
-
-    public override connectedCallback(): void {
-        super.connectedCallback();
-        this.forwardAriaLabelToSvg();
-    }
-
-    private ariaLabelChanged(): void {
-        this.forwardAriaLabelToSvg();
-    }
-
-    private forwardAriaLabelToSvg(): void {
-        const svg = this.shadowRoot?.querySelector('svg');
-        if (!svg) {
-            return;
-        }
-        if (this.ariaLabel !== null && this.ariaLabel !== undefined) {
-            svg.setAttribute('aria-label', this.ariaLabel);
-        } else {
-            svg.removeAttribute('aria-label');
-        }
-    }
 }
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
Do we still need to have `Icon` extend and use the `ARIAGlobalStatesAndProperties` mixin?

Reply:
I don't think so.